requirements.txt

```diff
@@ -1,3 +1,4 @@
 django-freeradius>=0.1.0a
 openwisp-users>=0.1.8,<0.2
 django-rest-auth>=0.9.3,<0.12.0
+django-cors-headers>=2.5.2
```

Comment:
this line makes `django-cors-headers` a dependency of `openwisp-radius` and will automatically install it when a developer using `openwisp-radius` will install this openwisp module.

But the user could set CORS in other ways, for example using Nginx. This is a configuration that is performed on project level, not on app-level (openwisp-radius is a django app, see the concept of `INSTALLED_APPS`, not a django project).

Instead of adding this here, install it in `.travis.yml` after `python setup.py install` is called.